### ISSUE TITLE
minor ui improvement: focus some "default" fields

### ISSUE THREAD

**Issue by hydrargyrum** [shaarli/Shaarli#1974]
When landing on the login page (`/login`), the username field should be automatically focused, or at least be accessible with a single `tab` press instead of the 8th press.

Also, when landing on the "shaare" page (`/admin/add-shaare`), the URL field should be focused.

And when clicking on the "search" header icon (the magnifier), at least one of the 2 search fields should be focused (let's say the leftmost one: text search).

**Comment by ArthurHoaro** [comment]
Thanks for the report! This was actually the case and got broken by another feature recently. 
I'm submitting a first quick fix for that.

I'm keeping this issue open because I like your suggestions:

  - [ ] autofocus search field in the dynamic header
  - [ ] review our "tab access" order for accessibility, especially for login page

**Comment by bttrx** [comment]
:+1:

**Comment by nodiscc** [comment]
@bttrx hi, please avoid :+1: -only comments as they notify everyone subscribed tot he issue or repository. Use the :+1: button in the first post, they are reflected [here](https://github.com/shaarli/Shaarli/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) (or even better, send a patch ;) )

**Comment by bttrx** [comment]
> send a patch

@nodiscc We'll see. No promise.
